src/Compilers/CSharp/Portable/CodeGen/EmitStatement.cs

```diff
@@ -1482,6 +1482,13 @@ private string GetLocalDebugName(ILocalSymbolInternal local, out LocalDebugId lo
                 return null;
             }
 
+            // Skip naming locals generated by instrumentation as they might not have a local syntax offset.
+            // Locals generated by instrumentation might exist in methods which do not contain a body (auto property initializers).
```

Comment:
Should this be combined with the check above which is validating that non user defined locals should not be named ( which includes ones generated by instrumentations)?